### PR TITLE
Prepare dev-v2.7-for-2.6.11

### PR DESCRIPTION
**Pull request by kinarashah**
- new branch `dev-v2.7-for-2.6.11` created from `release-v2.7` and rebased off of `dev-v2.7`.  
- resolved conflicts 
- removed 1.25 for rke1, rke2 and k3s

### CODE REVIEW

File: channels-rke2.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ appDefaults:
         defaultVersion: '1.22.x'
       - appVersion: '>= 2.6.5-0 < 2.6.7-0'
         defaultVersion: '1.23.x'
-      - appVersion: '>= 2.6.7-0 < 2.7.100-0'
+      - appVersion: '>= 2.6.7-0 < 2.7.2-0'
```
Comment (rancher-max) [inline channels-rke2.yaml]: I think this should be fine as the value is still greater than the current maximum Rancher version of 2.7.1. This should help with potential future merge conflicts, as this change was made to enable v1.25.x to be be `>= 2.7.2-0 < 2.7.100-0`, but calling it out here in this review just to be sure.

File: channels.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ appDefaults:
         defaultVersion: '1.22.x'
       - appVersion: '>= 2.6.5-0 < 2.6.7-0'
         defaultVersion: '1.23.x'
-      - appVersion: '>= 2.6.7-0 < 2.7.100-0'
+      - appVersion: '>= 2.6.7-0 < 2.7.2-0'
```
Comment (rancher-max) [inline channels.yaml]: Same as on the rke2 file, I think this should be fine as the value is still greater than the current maximum Rancher version of 2.7.1. This should help with potential future merge conflicts, as this change was made to enable v1.25.x to be be `>= 2.7.2-0 < 2.7.100-0`, but calling it out here in this review just to be sure.

**Review by rancher-max — DISMISSED** [review]

**Review by snasovich — DISMISSED** [review]
LGTM, reviewed diff between `v2.7-dev` branch and source branch to confirm that the differences are limited to removal of v1.25 k8s versions: https://github.com/rancher/kontainer-driver-metadata/compare/dev-v2.7..kinarashah:kontainer-driver-metadata:v27

However, I do agree with @rancher-max 's comment about changing `- appVersion: '>= 2.6.7-0 < 2.7.2-0'` to `- appVersion: '>= 2.6.7-0 < 2.7.100-0'` for `1.24.x` for RKE2/k3s as there is a chance 1.25 won't be default in 2.7.2 and it's safer to retain < 2.7.100-0 there.

**Comment by kinarashah** [comment]
@rancher-max @snasovich Thank you for reviewing, I've updated the range for 1.24 rke2/k3s back to `'>= 2.6.7-0 < 2.7.100-0'`

**Review by rancher-max — APPROVED** [review]
LGTM for k3s/rke2! Thanks Kinara!

**Review by snasovich — APPROVED** [review]
LGTM!

**Pull request merged**